javascripts/govuk/show-hide-content.js

```diff
@@ -17,7 +17,7 @@
     // Escape name attribute for use in DOM selector
     function escapeElementName (str) {
       var result = str.replace('[', '\\[').replace(']', '\\]')
-      return (result)
```

Comment:
We could just return the above expression, result isn't used anywhere else so just keep it simple..